cmd/cluster-agent/app/clusterchecks.go

```diff
@@ -8,47 +8,8 @@
 
 package app
 
-import (
-	"fmt"
-
-	"github.com/fatih/color"
-	"github.com/spf13/cobra"
-
-	"github.com/DataDog/datadog-agent/cmd/agent/common"
-	"github.com/DataDog/datadog-agent/pkg/config"
-	"github.com/DataDog/datadog-agent/pkg/flare"
-)
+import "github.com/DataDog/datadog-agent/cmd/cluster-agent/commands"
```

Comment:
don't you think you should share the ownership of that pkg since it's used by both cluster agents?